packages/web/src/services/AudiusBackend.ts

```diff
@@ -784,6 +852,12 @@ class AudiusBackend {
     sort = null,
     filterDeleted = null,
     withUsers = true
+  }: Omit<
+    DiscoveryAPIParams<typeof DiscoveryAPI.getTracks>,
+    'sort' | 'filterDeleted'
+  > & {
+    sort: Nullable<boolean>
```

Comment:
not sure if it's exactly what you want, but could try `SetOptional` from 'type-fest', it will optionalize sort and filterDeleted?

Reply:
That is basically what I wanted but type-fest doesn't have a `SetNullable` from what I can tell. Also we don't have it installed in web yet, but should add it